schemas/clinvar/tables/ClinicalAssertion.json

```
@@ -0,0 +1,103 @@
+ {
+    "name": "clinical_assertion",
+    "columns": [
+	{
+	    "name": "clinical_assertion_id",
```

Here and elsewhere, I think it'd help clarity to name the primary ID column `id` in each table, instead of having a variable name. That'll make it easy to distinguish primary IDs from foreign ID references when scanning the column names.
```suggestion
	    "name": "id",
```

---

Agree.